climada/hazard/test/test_tc_tracks.py

```diff
@@ -308,6 +308,26 @@ def test_read_legacy_netcdf(self):
             self.assertEqual(tr.basin.shape, tr.time.shape)
             np.testing.assert_array_equal(tr.basin, "SP")
 
+    def test_hdf_io(self):
+        """Test writting and reading hdf5 TCTracks instances"""
+        path = DATA_DIR.joinpath("tc_tracks.h5")
+        tc_track = tc.TCTracks.from_ibtracs_netcdf(
+            provider='usa', year_range=(1993, 1994), basin='EP', estimate_missing=True)
+        tc_track.write_hdf(path)
+        tc_read = tc.TCTracks.from_hdf(path)
+
+        self.assertEqual(len(tc_track.data), len(tc_read.data))
+        for tr, tr_read in zip(tc_track.data, tc_read.data):
+            self.assertEqual(set(tr.attrs.keys()), set(tr_read.attrs.keys()))
+            self.assertEqual(set(tr.variables), set(tr_read.variables))
+            self.assertEqual(set(tr.coords), set(tr_read.coords))
+            for key in tr.attrs.keys():
+                self.assertEqual(tr.attrs[key], tr_read.attrs[key])
+            for v in tr.variables:
+                self.assertEqual(tr[v].dtype, tr_read[v].dtype)
+                np.testing.assert_array_equal(tr[v].values, tr_read[v].values)
+            self.assertEqual(tr.sid, tr_read.sid)
+
```

Comment:
```suggestion
path.unlink()
```
Maybe remove the file after the test?

Reply:
Good idea!